hijack/tests/test_hijack.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.test import TestCase, Client, RequestFactory
+from django.test import TestCase, Client, RequestFactory, override_settings
```

Comment:
`override_settings`
imported but not used

Reply:
Good catch, we should probably add some linters to the CI suite.

Reply:
Agree about linters